Source/BluemixAppID/api/UserAttributeDelegate.swift

```diff
@@ -11,7 +11,6 @@ import Foundation
 
 public protocol UserAttributeDelegate {
```

Comment:
UserAttributes APIs are simple APIs that return true/false. We should use completionHandlers instead of delegates.